tasks/deploy.py

```diff
@@ -20,6 +20,7 @@ def deploy(
     stack_name: Optional[str] = None,
     pipeline_id: Optional[str] = None,
     install_agent: Optional[bool] = None,
+    install_docker: Optional[bool] = False,
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
It makes sense to have it here, even if currently no code is using it, we might add it as a VM option in the future